mitodl/open-discussions
# Make sure object_type is always merged in when retrieving entities from state

#### Pre-Flight checklist


#### What are the relevant tickets?
Closes #2228

#### What's this PR do?
Merges the object_type property with appropriate value into entities retrieved from state.

#### How should this be manually tested?
- Click a new course on the index page to bring up the drawer. Close it and repeat.  The drawer should open every time.
- Add some favorite courses and bootcamps, and click on them too in the Favorites carousel.
